examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -168,7 +168,7 @@ message ABitOfEverything {
 
 	string uuid = 1 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {pattern: "[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}", min_length: 1}];
 	repeated Nested nested = 2;
-	float float_value = 3;
+	float float_value = 3 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {description: "Float value field", default: "0.2", required: ['float_value']}];
```

Comment:
I'm a little confused about this taking an array of "required" parameters. What does it mean in this case?

Reply:
I assume it's this way because when the field is a _Message_ type you are able to specify several of them? Could we try to make an example where several are of them are specified?

Reply:
Sure I can but inside openapiv2_field It won't have any effect because the field is translated to a boolean value (here)[https://github.com/grpc-ecosystem/grpc-gateway/pull/852/files#diff-d419038d63db0fdabc61f0d6cc6683aeR125]. openapiv2_field and openapiv2_schema use (swaggerSchemaObject)[https://github.com/grpc-ecosystem/grpc-gateway/blob/master/protoc-gen-swagger/genswagger/types.go#L200] as schema, that's why on both cases it has to be managed as array but only on openapiv2_schema has any sense to use it as a multiple value array. On  openapiv2_field is used as array due to the actual implementation but it will have more sense to used it as a boolean.

Reply:
I see, nevermind then. It feels a little bit strange though, how much work would it be to make it a boolean instead?

Reply:
Isn't there somewhere we can specify multiple `required` for a Message still? I understand the field uses an array for other reasons, but it looks like the swagger definition expects an array, and specifies an array. Can we find one such example?